target_chains/solana/pyth_solana_receiver_sdk/src/lib.rs

```diff
@@ -1,7 +1,15 @@
-use anchor_lang::declare_id;
+use {
+    anchor_lang::declare_id,
+    solana_program::{
+        pubkey,
+        pubkey::Pubkey,
+    },
+};
 
 pub mod config;
 pub mod error;
 pub mod price_update;
 
 declare_id!("rec5EKMGg6MxZYaMdyBfgwp4d5rB9T1VQH5pJv5LtFJ");
+
+pub const PYTH_PUSH_ORACLE_ID: Pubkey = pubkey!("F9SP6tBXw9Af7BYauo7Y2R5Es2mpv8FP5aNCXMihp6Za");
```

Comment:
This is confusing to me. You have a separate sdk.rs above for the push oracle which has some useful methods in it. It seems like there should either be one sdk for users of both push/pull, or two different ones. At the moment, it seems like a mix of both options (?)

Reply:
- `pyth-solana-receiver-sdk` is for consuming price updates
- `pyth_solana_receiver::sdk` is for posting price updates 
- `pyth_push_oracle::sdk` is for updating price feeds

Reply:
For this program id, I need to put it here because otherwise there are some circular dependencies that break the code.